app/controllers/cart_items_controller.rb

```diff
@@ -11,8 +11,10 @@ def create
   end
 
   def destroy
-    @cart.contents.delete_if { |key, value| key == params[:id] }
-
+    item = Item.find(params[:id].to_i)
+    if @cart.contents.delete_if { |item_id, quantity| item_id == params[:id] }
```

Comment:
Unused block argument - `quantity`. If it's necessary, use `_` or `_quantity` as an argument name to indicate that it won't be used.